packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -95,7 +95,28 @@ const mapAttachmentType = (attachmentType: AttachmentType): FileMetadataAttachme
   return 'unknown';
 };
 
-/* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images) */
+const processTeamsImageContent = (message: ChatMessageWithStatus): string | undefined => {
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
I think you can move this conditional compile out above the `const`. I'm actually a little surprised the stable build didn't throw an error saying this was unused. Maybe cause this is ` | undefined`?

Reply:
Oh I see in the stable it wold always just return `message.content?.message` which is what it used to do anyways...ok so this is fine

Reply:
We can't move this outside the const because `processTeamsImageContent` is being called in stable
<img width="357" alt="image" src="https://github.com/Azure/communication-ui-library/assets/77021369/3a70f9dd-1fc1-4bb3-863a-3e43097d6901">
But will rename it to something more generic so conditional-compile-remove will make sense